modules/behavioural_qc/php/views/conflicts.class.inc

```diff
@@ -0,0 +1,58 @@
+<?php declare(strict_types=1);
+
+namespace LORIS\behavioural_qc\Views;
+
+/**
+ * This class formats a form object into array for the behavioural_qc module
+ *
+ * @category ConflictsViews
+ * @package  Loris
+ * @author   Alizee Wickenheiser <alizee.wickenheiser@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+class Conflicts
+{
+    /**
+     * The formElement
+     *
+     * @var array
+     */
+    private $_formElement = [];
+
+    /**
+     * Constructor
+     *
+     * @param array $visits      The visits for form element
+     * @param array $instruments The instruments for form element
+     * @param array $projects    The projects for form element
+     * @param array $sites       The sites for form element
+     */
+    public function __construct($visits, $instruments, $projects, $sites)
+    {
+        $this->_formElement = [
+            'instruments'       => $instruments,
+            'candID'            => null,
+            'pscID'             => null,
+            'visits'            => $visits,
+            'projects'          => $projects,
+            'sites'             => $sites,
+            'fieldName'         => null,
+            'tableName'         => null,
+            'test_name_display' => null,
+            'sessionID'         => null,
+            'commentid'         => null,
+        ];
+    }
+
+    /**
+     * Generates the array
+     *
+     * @return array
+     */
+    public function toArray() : array
+    {
+        return $this->_formElement;
+    }
+}
```

Comment:
I'm a little confused by these View (`Conflicts`, `Behavioural`, `Incomplete`) classes, could you give me some context on them? Are they used elsewhere in LORIS? Is this specific to this PR? Why do they constitute a class?

Reply:
@HenriRabalais I can probably get rid of these classes if you think it's best? I know it originally came about from chatting with @xlecours about the best approach I should take to use the Dataframework which is part of this PR. That's why there are 3 directories named: models, provisioners, and views.